************* Module multiprocessing.sharedctypes
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/sharedctypes.py:44:0: C0103: Function name "RawValue" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/sharedctypes.py:54:0: C0103: Function name "RawArray" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/sharedctypes.py:59:4: R1705: Unnecessary "else" after "return" (no-else-return)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/sharedctypes.py:70:0: C0103: Function name "Value" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/sharedctypes.py:84:0: C0103: Function name "Array" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/sharedctypes.py:107:4: R1705: Unnecessary "elif" after "return" (no-else-return)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/sharedctypes.py:107:23: W0212: Access to a protected member _SimpleCData of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/sharedctypes.py:110:11: W0212: Access to a protected member _type_ of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/sharedctypes.py:118:43: W0212: Access to a protected member _fields_ of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/sharedctypes.py:119:12: C0103: Variable name "d" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/sharedctypes.py:130:4: R1705: Unnecessary "else" after "return" (no-else-return)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/sharedctypes.py:131:31: W0212: Access to a protected member _type_ of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/sharedctypes.py:131:43: W0212: Access to a protected member _wrapper of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/sharedctypes.py:131:57: W0212: Access to a protected member _length_ of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/sharedctypes.py:133:42: W0212: Access to a protected member _wrapper of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/sharedctypes.py:141:4: W0212: Access to a protected member _wrapper of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/sharedctypes.py:152:8: C0103: Variable name "d" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/sharedctypes.py:153:8: W0122: Use of exec (exec-used)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/sharedctypes.py:157:0: C0103: Constant name "template" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/sharedctypes.py:180:0: R0205: Class 'SynchronizedBase' inherits from object, can be safely removed from bases in python3 (useless-object-inheritance)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/sharedctypes.py:187:25: E1123: Unexpected keyword argument 'force' in method call (unexpected-keyword-arg)

-----------------------------------
Your code has been rated at 8.18/10

